.gitignore

```diff
@@ -179,3 +179,8 @@ $RECYCLE.BIN/
 
 # Visual Studio Code
 .vscode/
+
+# JetBrains Rider
+.idea/
+*.DotSettings
```

Comment:
❔ Should this go under the ReSharper settings above? I vaguely remember seeing people put it there in the past, but I could be wrong and/or things could have changed.